### PR TITLE
Kp 5580 asr tests

### PR DESCRIPTION
**Pull request by mmatthiesencsc**
Adding this code should not hurt, but the whole process of updating aalto-asr is not well defined.

### CODE REVIEW

File: commandline/roles/aalto-asr/README.md [hunk 1]
```diff
@@ -3,4 +3,21 @@ aalto-asr
...
 
+# Install for local testing as unpriviledged user
 
+`cd Kielipankki-palvelut/commandline`
+
+Install as normal user to puhti-login2:
+`ansible-playbook site.yml -vv -i inventories/test -t aalto-asr`
+
+Login to puhti-login2.csc.fi:
+`ssh puhti-login2.csc.fi`
```
Comment (aajarven) [inline commandline/roles/aalto-asr/README.md:14]: The current test inventory uses node 12, so these are outdated

**Review by aajarven — CHANGES_REQUESTED** [review]
This PR seems to be of a good vintage. At least the login node has changed since, and I'm not too confident that that is the only thing. Perhaps we should do `git rebase origin/main`, resolve conflicts (if needed) and verify that this playbook works before merging?

**Comment by mmatthiesencsc** [comment]
Hi Anni, thanks for making me think twice! This is now tested and working. I did not re-install again, but I did successfully run  ansible-playbook site.yml -vv -i inventories/test_csc -t aalto-asr. There was indeed a bit more to make it work than I originally thought.

**Review by aajarven — APPROVED** [review]
Looking good!

**Pull request merged**